docs/rules/no-prototype-builtins.md

```diff
@@ -0,0 +1,47 @@
+# Disallow use of Object.prototypes builtins directly (no-prototype-builtins)
+
+In ECMAScript 5.1, `Object.create` was added, which enables the creation of objects with a specified `[[Prototype]]`. `Object.create(null)` is a common pattern used to create objects that will be used as a Map. This can lead to errors when it is assumed that objects will have properties from `Object.prototype`. This rule prevents calling `Object.prototype` methods directly from an object.
+
+## Rule Details
+
+This rules disallows use of `Object.prototype` builtins directly on object instances.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint no-prototype-built-ins: "error"*/
+
+var foo = Object.create(null);
+var hasBarProperty = foo.hasOwnProperty("bar");
+
+var foo = Object.create(null);
+var bar = {};
+var isPrototypeOfBar = foo.isPrototypeOf(bar);
+
+var foo = Object.create(null);
+var barIsEnumerable = foo.propertyIsEnumerable("bar");
+```
+
+Examples of **correct** code for this rule:
+
+```js
+/*eslint no-prototype-built-ins: "error"*/
+
+var foo = Object.create(null);
+var hasBarProperty = Object.prototype.hasOwnProperty.call(foo, "bar");
```

Comment:
Mind providing an example of what you mean? In this case, `{}.method` would be an example of incorrect code for this rule. Assuming I'm just misunderstanding the ask.